### PR TITLE
HOT-1907 Clear the pending list within Create_Person_Complete rather …

### PR DESCRIPTION
**Pull request by NagaPradeepGajula-osi**
…than delete

### Jira Story

- [Name of Story HOT-1907](https://osi-cwds.atlassian.net/browse/HOT-1907)

## Tests
- [x] I have included unit tests 
- [ ] I have included feature tests 
- [ ] I have included other tests 
- [ ] I have NOT included tests 

## Types of changes

- [ ] New feature (non-breaking change which adds functionality)
- [x] Refactoring (No behavioral changes)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the code style of this project.
- [x] I have ran all tests for the project.
- [x] I have ran lint/rubocop check for the changed/new files.
- [x] My code is in a stable state ready to be deployable, but not necessarily complete.
- [x] I promise on my honor as a CWDS developer to ensure I don't break the build, to check Lint/Rubocop, use best practices, to leave the code in better shape than I found it, and to have a good day.

### CODE REVIEW

File: spec/javascripts/reducers/pendingParticipantsReducerSpec.js [hunk 1]
```diff
@@ -28,26 +26,12 @@ describe('pendingParticipantsReducer', () => {
       const secondId = '3'
       const oldState = fromJS([firstId, secondId])
       const action = createPersonSuccess({id: secondId})
-      const expectedState = fromJS([firstId, secondId])
+      const expectedState = fromJS([firstId])
```
Comment (wmitchell-cambria) [inline spec/javascripts/reducers/pendingParticipantsReducerSpec.js]: Oh, wow. Good catch here. This was actually asserting the opposite of what the test described.

**Comment by NagaPradeepGajula-osi** [comment]
yes I did check it manually

File: spec/javascripts/selectors/screening/relationshipsSelectorsSpec.js [hunk 2]
```diff
@@ -99,14 +99,14 @@ describe('relationshipsViewSelectors', () => {
         {
           name: 'Ricky Robinson',
           relationships: [
-            {relatee: 'Johny Robinson', legacy_descriptor: {legacy_id: '2'}, type: 'Brother', person_card_exists: false},
+            {relatee: 'Johny Robinson', legacy_descriptor: {legacy_id: '2'}, type: 'Brother', person_card_exists: true},
```
Comment (wmitchell-cambria) [inline spec/javascripts/selectors/screening/relationshipsSelectorsSpec.js]: These tests don't set any participants in the state, so I would expect `person_card_exists` to end up as false. Looking at the logic in `isPersonCardExists` and in `Relationships.jsx`, it looks like we are actually treating it as a test of whether the person card does *not* exist.

So, it's working, but we've flipped our trues and falses the whole way through.

**Comment by NagaPradeepGajula-osi** [comment]
retest this please

**Comment by NagaPradeepGajula-osi** [comment]
retest this please

**Pull request merged**